RepoCleanup/Application/CommandHandlers/DeleteRepoForOrgsCommandHandler.cs

```diff
@@ -0,0 +1,53 @@
+﻿using RepoCleanup.Application.Commands;
+using RepoCleanup.Models;
+using RepoCleanup.Services;
+using System.Net;
+using System.Threading.Tasks;
+
+namespace RepoCleanup.Application.CommandHandlers
+{
+    public class DeleteRepoForOrgsCommandHandler
+    {
+        private readonly GiteaService _giteaService;
+        public DeleteRepoForOrgsCommandHandler(GiteaService giteaService)
+        {
+            _giteaService = giteaService;
+        }
+
+        public async Task<int> Handle(DeleteRepoForOrgsCommand command)
+        {
+            var reposDeletedCounter = 0;            
+
+            foreach (var org in command.Orgs)
+            {
+                var repoName = command.PrefixRepoNameWithOrg ? $"{org}-{command.RepoName}" : command.RepoName;
+
+                var giteaResponse = await _giteaService.GetRepo(org, repoName);
+                if(giteaResponse.StatusCode == HttpStatusCode.OK)
+                {
+                    giteaResponse = await _giteaService.DeleteRepository(org, repoName);
+                    
+                    if(giteaResponse.Success)
+                    {                        
+                        reposDeletedCounter++;
+                    }
+                }                
+            }
+
+            return reposDeletedCounter;
+        }
+
+        private CreateRepoOption GetCreateRepoOption(string repoName)
```

Comment:
Method not in use?

Reply:
Yes, I noticed it. Coming a fix in the next commit. Started to look into Mediator pattern during the weekend to get some experience with it, so my branch isn't in a committable state just yet.